firmware/application/transmitter_model.hpp

```diff
@@ -26,77 +27,68 @@
 #include <cstdint>
 #include <cstddef>
 
-#include "receiver_model.hpp"
 #include "app_settings.hpp"
+#include "max2837.hpp"
 #include "message.hpp"
+#include "receiver_model.hpp"
 #include "rf_path.hpp"
-#include "max2837.hpp"
-#include "volume.hpp"
 #include "signal.hpp"
 
 class TransmitterModel {
    public:
+    struct settings_t {
+        uint32_t baseband_bandwidth = max283x::filter::bandwidth_minimum;
+        uint32_t sampling_rate = 3'072'000;
+        uint32_t channel_bandwidth = 1;
+        /* 35 should give approx 1m transmission range. */
+        uint8_t tx_gain_db = 35;
+        bool rf_amp = false;
+    };
+
     /* The frequency to transmit on. */
     rf::Frequency target_frequency() const;
     void set_target_frequency(rf::Frequency f);
 
-    void set_antenna_bias();
-
-    bool rf_amp() const;
-    void set_rf_amp(bool enabled);
-
-    // TODO: does this make sense on TX?
-    int32_t lna() const;
```

Comment:
I finally deleted LNA and VGA from transmitter_model. Nothing was actually using them.
There's a _slight_ possibility that there was some impact on the radio state (because update_lna/vga no longer getting called). A simple TX test should be sufficient.